### PR TITLE
Clean up work package's action menu

### PR DESCRIPTION
**Pull request by hschink**
Add missing planning element actions to work package's action menu.

This PR relates to https://www.openproject.org/issues/1424

### CODE REVIEW

File: features/issues/issue_edit.feature [hunk 1]
```diff
@@ -75,8 +75,8 @@ Feature: Issue edit
...
-    And I should not see "% Done" within "#issue-form"
-    And there should be the disabled "#issue_priority_id" element within "#issue-form"
-    And there should be the disabled "#issue_start_date" element within "#issue-form"
-    And there should be the disabled "#issue_due_date" element within "#issue-form"
-    And there should be the disabled "#issue_estimated_hours" element within "#issue-form"
+    And I should not see "% Done" within "#work_package-form"
+    And there should be the disabled "#work_package_priority_id" element within "#work_package-form"
+    And there should be the disabled "#work_package_start_date" element within "#work_package-form"
+    And there should be the disabled "#work_package_due_date" element within "#work_package-form"
+    And there should be the disabled "#work_package_estimated_hours" element within "#work_package-form"
```
Comment (apfelfabrik) [inline features/issues/issue_edit.feature]: There probably should not be any IDs in the cukes. The previous selectors failed now hat IDs changed, perhaps they should be removed altogether?

**Comment by apfelfabrik** [comment]
Duplicate does not lead to the work_package show action after completion. Is this out of scope for this issue?

**Comment by apfelfabrik** [comment]
Delete removes the work package and does not trash it. It this out of scope for this issue?

**Comment by apfelfabrik** [comment]
Some of the features fail for me after merging w/ the feature/rails3 branch.

<pre>
Failing Scenarios:
cucumber features/work_packages/editable_fields.feature:27 # Scenario: Going to the page and viewing all the fields
cucumber features/work_packages/preview.feature:36 # Scenario: Previewing a new work package
cucumber features/issues/issue_edit.feature:42 # Scenario: User updates an issue successfully
</pre>

**Comment by ulferts** [comment]
Duplicate and trash is out of the scope, yes. Duplicate is part of #294, while trash is not yet developed. It would be very easy to enable it but I can not fathom what that will entail. We will have to at least scope a lot of selectors for views and that in itself is not to be done in the passing.

I will take a look at the IDs. Shouldn't be to hard.

**Comment by ulferts** [comment]
I removed the usage of ids from the cukes @apfelfabrik mentioned.

The tests are green on our CI.

**Comment by apfelfabrik** [comment]
Broken cukes were my fault, sorry for that.

**Pull request merged**